terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/managereservation/ManageReservationHelper.java

```diff
@@ -62,6 +64,9 @@
     @Inject
     Mapper dozerBeanMapper;
 
+    @Value("${reservation.report.pdf.name}")
+    String reservationReportPdfName;
```

Comment:
78a85d1fa6add363af9cc8f53121cb12c248fe91

I fixed in above commit .

Reply:
It hasn't been modified as your comment...
PDF name should be obtained from controller. Because generally file name contains unique name (reservation number, reservation date, reservation owner, etc...)

If you want to set PDF name in `prepareResponse`, you must hold it with request.
I think that it may be set in `mergePdfDocument` (think the file name is a part of PDF).